tesseract/tesseract_collision/include/tesseract_collision/bullet/bullet_utils.h

```diff
@@ -341,8 +335,8 @@ inline btScalar addDiscreteSingleResult(btManifoldPoint& cp,
 {
   assert(dynamic_cast<const CollisionObjectWrapper*>(colObj0Wrap->getCollisionObject()) != nullptr);
   assert(dynamic_cast<const CollisionObjectWrapper*>(colObj1Wrap->getCollisionObject()) != nullptr);
-  const CollisionObjectWrapper* cd0 = static_cast<const CollisionObjectWrapper*>(colObj0Wrap->getCollisionObject());
-  const CollisionObjectWrapper* cd1 = static_cast<const CollisionObjectWrapper*>(colObj1Wrap->getCollisionObject());
+  const auto* cd0 = static_cast<const CollisionObjectWrapper*>(colObj0Wrap->getCollisionObject());
```

Comment:
This is one case where I like the use of auto. The type is really obvious. It saves space.